### PR TITLE
Add galactic coords option to lt_radec

### PR DESCRIPTION
**Pull request by profxj**
As titled.

### CODE REVIEW

File: linetools/scripts/lt_radec.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def parser(options=None):
     parser = argparse.ArgumentParser(
         description='Print coordinates in several formats from input one. [v1.1]')
     parser.add_argument("inp", nargs='?', default=None, help="RA,DEC (e.g. 152.25900,7.22885), JXX (e.g. J100902.16+071343.8)")
+    parser.add_argument("-g", "--gal", default=False, action='store_true', help="Input is (l,b)")
```
Comment (ntejos) [inline linetools/scripts/lt_radec.py]: `Input is Galactic (l,b) in degrees.`

File: linetools/scripts/lt_radec.py [hunk 2]
```diff
@@ -40,19 +41,20 @@ def main(args=None):
...
     print('   ')
-    print('   {:s} {:s}   (J2000)'.format(coord.ra.to_string(unit=u.hour,sep=':',pad=True),
-                             coord.dec.to_string(sep=':',pad=True,alwayssign=True)))
-    print('   RA={:f} deg, DEC={:f} deg'.format(coord.ra.deg, coord.dec.deg))
-    print('   radec = ({:f},{:f}) deg'.format(coord.ra.deg, coord.dec.deg))
+    print('   {:s} {:s}   (J2000)'.format(coord.icrs.ra.to_string(unit=u.hour,sep=':',pad=True),
+                             coord.icrs.dec.to_string(sep=':',pad=True,alwayssign=True)))
+    print('   RA={:f} deg, DEC={:f} deg'.format(coord.icrs.ra.deg, coord.icrs.dec.deg))
+    print('   radec = ({:f},{:f}) deg'.format(coord.icrs.ra.deg, coord.icrs.dec.deg))
+    print('   galactic = ({:f},{:f}) deg'.format(coord.galactic.l.deg, coord.galactic.b.deg))
```
Comment (ntejos) [inline linetools/scripts/lt_radec.py]: galactic -> Galactic

File: linetools/tests/test_utils.py [hunk 3]
```diff
@@ -134,6 +134,9 @@ def test_radeccoord():
     # List
     coords = ltu.radec_to_coord(lcoord)
     assert len(coords) == 6
+    # Galactic
+    gcoord = ltu.radec_to_coord((280.5,-32.9), gal=True)
+    assert np.isclose(gcoord.icrs.ra.value, 80.8456130588062)
```
Comment (ntejos) [inline linetools/tests/test_utils.py]: add test for expected declination too?

File: linetools/utils.py [hunk 4]
```diff
@@ -192,16 +192,20 @@ def radec_to_coord(radec):
         ('12:45:11','14:45:23')  -- Assumes positive DEC
         (123.123, 12.1224) -- Assumed deg
         [(123.123, 12.1224), (125.123, 32.1224)]
+    gal : bool, optional
+      Input pair of floats are l,b in deg
```
Comment (ntejos) [inline linetools/utils.py]: (l,b)

File: linetools/utils.py [hunk 5]
```diff
@@ -212,10 +216,13 @@ def radec_to_coord(radec):
             else:
                 DEC = radec[1]
             #
-            coord = SkyCoord(radec[0]+DEC, frame='icrs',
+            coord = SkyCoord(radec[0]+DEC, frame=frame,
```
Comment (ntejos) [inline linetools/utils.py]: this may give the wrong coordinate if `frame=='galactic'`... although not sure.

**Comment by profxj** [comment]
@yzhenggit -- The 'close' button is for a PR that you do *not* want to approve.  :)
Yes, this stuff is confusing..

File: linetools/scripts/lt_radec.py (hunk 1 above)
Comment (profxj) [reply linetools/scripts/lt_radec.py]: modified

File: linetools/scripts/lt_radec.py (hunk 2 above)
Comment (profxj) [reply linetools/scripts/lt_radec.py]: fixed

File: linetools/tests/test_utils.py (hunk 3 above)
Comment (profxj) [reply linetools/tests/test_utils.py]: added

File: linetools/utils.py (hunk 4 above)
Comment (profxj) [reply linetools/utils.py]: fixed

**Comment by yzhenggit** [comment]
Oops... my fault ...

File: linetools/utils.py (hunk 5 above)
Comment (profxj) [reply linetools/utils.py]: i think we are ok here.

**Comment by profxj** [comment]
merging

**Pull request merged**